.changeset/small-rabbits-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/create-hydrogen': patch
+---
+
+Separate create-hydrogen template question into template and language questions.
```

Comment:
@cartogram looks like a possible major?
If someone coded commands to be `dev create-hydrogen --path ../cli-tests --template hello-world-ts` it won't work anymore, they will have to use:
```
dev create-hydrogen --path ../cli-tests --template hello-world --ts
```

Reply:
thoughts?